spec/requests/orders_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+RSpec.describe "OrdersControllers", type: :request do
+  let!(:customer) { FactoryBot.create(:customer) }
+
+  describe "get orders_path" do
+    it "renders the index view" do
+      FactoryBot.create_list(:order, 10, customer: customer)
+      get orders_path
+      expect(response).to render_template(:index)
```

Comment:
When testing a controller, you generally want to check the status code. While `render_template` is taught in this lesson, it's deprecated because that tests the application's implementation rather than behaviour.